R/evalParsRollapply.R

```diff
@@ -18,7 +18,7 @@ evalParsTS <- function(par.matrix, object,
   }
 
   # Sets default settings for parallelisation if missing
-  parallel <- hydromad.parallel(parallel)
+  parallel <- hydromad:::hydromad.parallel(parallel)
```

Comment:
```suggestion
  parallel <- hydromad.parallel(parallel)
```
::: was probably needed for your testing, but not within the package - R CMD CHECK will give a warning about this

Reply:
fair enough I can remove this

Reply:
removed in 2521115